.wordpress-org/blueprint/blueprint.json

```diff
@@ -0,0 +1,16 @@
+{
+	"landingPage": "\/wp-admin\/options-general.php?page=antispam_bee",
+	"steps": [
+		{
+			"step": "login",
+			"username": "admin",
+			"password": "password"
+		},
+		{
+			"step": "defineWpConfigConsts",
+			"consts": {
+			  "IS_PLAYGROUND_PREVIEW": true
```

Comment:
Not sure if we should prefix that, to make it less likely that the constant is added by something else

Reply:
This is only added on Playground. Network is not activated, so you can't add another plugin.